css/styles.css

```diff
@@ -65,7 +65,9 @@ body {
   font-size: 3em;
 }
 
-#testimonial-text {
+#testimonial-text-nic,
```

Comment:
Why not just make this a class instead of separating it?

Reply:
Changed to a class